src/librustc_mir/lib.rs

```diff
@@ -39,6 +39,7 @@ Rust MIR: a lowered representation of Rust. Also: an experiment!
 #![feature(collection_placement)]
 #![feature(nonzero)]
 #![feature(underscore_lifetimes)]
+#![feature(crate_visibility_modifier)]
```

Comment:
Was this needed?

Reply:
yes, due to the usage here `crate trait FindAssignments {`